portal-api/api-tests/v1/deals/deals.api-test.js

```diff
@@ -32,14 +35,13 @@ describe('/v1/deals', () => {
   let noRoles;
   let anHSBCMaker;
   let aBarclaysMaker;
-  let aBarclaysChecker;
   let aSuperuser;
+  let testUsers;
 
   beforeAll(async () => {
-    const testUsers = await testUserCache.initialise(app);
+    testUsers = await testUserCache.initialise(app);
     noRoles = testUsers().withoutAnyRoles().one();
     aBarclaysMaker = testUsers().withRole('maker').withBankName('Barclays Bank').one();
```

Comment:
I'll stop commenting "constants" now -- might be worth a find all :)

Reply:
Yep thank you! I had forgotten that I'd only used the constants in `src` and not `api-tests`. Should be fixed now!